packages/room/config/config.js

```diff
@@ -20,19 +20,31 @@ export const webrtc = {
 }
 
 export const media = {
-  video: {
-    width: 1280,
-    height: 720,
-    frameRate: 24,
-    facingMode: 'user',
-    aspectRatio: 1.777_777_777_8,
-    maxBitrate: 1_200_000,
-    codec: 'VP9',
+  webcam: {
+    maxFramerate: 30,
+    videoCodecs: ['video/VP9'],
+    simulcast: true,
+    svc: true,
+    bitrate: {
+      highBitrate: 1200 * 1000,
+      midBitrate: 500 * 1000,
+      lowBitrate: 150 * 1000,
+    },
+    scalabilityMode: 'L3T1',
+  },
+  screen: {
+    maxFramerate: 30,
+    videoCodecs: ['video/VP8'],
```

Comment:
The order of codecs can be different with the camera config, but the list of codecs must contain the same codecs. So this will be VP8,H264,VP9

Reply:
This was also updated on this commit [09f0fc3](https://github.com/inlivedev/inlive-js-sdk/pull/96/commits/09f0fc34a60a17f914bdbf21a4c204a68243a7b7).